corehq/util/django_migrations.py

```diff
@@ -56,6 +56,17 @@ def add_if_not_exists_raw(string, name):
     ])
 
 
+def add_if_exists_raw(string, name):
```

Comment:
nit: maybe a better name
```suggestion
def execute_sql_if_exists_raw(string, name):
```